pkg/acs/calacs/History

```diff
@@ -1,3 +1,6 @@
+###  01-Sep-2019 - MDD -- Version 10.2.1
+    Removed/modified/updated routines which used code from numerical recipes. 
```

Comment:
Would this be better stated "used copyrighted code"?

Reply:
Done